### DISCUSSION TITLE
[BUG] Creating multiple conversations on the same lines during code review

### DISCUSSION THREAD

**Discussion by domiins** [community/community#7477] [Pull Requests] [answered]
When creating code comments during code review, it is unable to create a new conversation on the part of code that already has an open conversation.

- For single-line conversations:
  - When clicking the `+` comment icon, I am redirected to add a reply to the already open unresolved conversation.

- For multiple-line conversations:
  - When selecting the same range of lines that already have a conversation started and then clicking the `+` button, nothing happens.
  - It is possible to create a new conversation only when not selecting all of the lines.

The expected behavior for both cases is to start a new conversation instead of the current behavior.
Basically to behave the same way in case there is a conversation already open or not.

**Environment**
App: Web app
Browser: Google Chrome 92.0.4515.159 / Mozilla Firefox 94.0

**Comment by domiins** [comment, marked as answer]
I guess this is fixed and solved now.
